chromium_src/third_party/tflite_support/sources.gni

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//third_party/flatbuffers/flatbuffer.gni")
+
+text_embedder_fix_sources = [
```

Comment:
can we create a separate target for this in brave to avoid patching?

Reply:
Yes and that method is better than patching.
Fixed in https://github.com/brave/brave-core/pull/24953/commits/9d4da565d2b06599bea4ef9e6c1c365f7963d3f4